Entity/AbstractEntity.php

```diff
@@ -28,6 +28,16 @@ public function getCreated()
     }
 
     /**
+     * Set updated date
+     *
+     * @param \Datetime $date The updated date
+     */
+    public function setUpdated($date)
```

Comment:
Actually we intentionally skipped this function in this Entity because it is automatically done by gedmo